src/medImageIO/itkDCMTKImageIO.cpp

```diff
@@ -52,6 +52,9 @@ double DCMTKImageIO::MAXIMUM_GAP = 999999;
 
 DCMTKImageIO::DCMTKImageIO()
 {
+    // Avoid to display log of each DICOM metadata not read by itkMetaImageIO
+    this->SetGlobalWarningDisplay(false);
```

Comment:
This line avoids to display the itkWarning messages at import (at least, i'll have to test again in pipelines)